invenio_users_resources/services/groups/service.py

```diff
@@ -56,3 +58,100 @@ def rebuild_index(self, identity, uow=None):
         self.indexer.bulk_index([r.id for r in roles])
 
         return True
+
+    @unit_of_work()
+    def create(self, identity, data, raise_errors=True, uow=None):
+        """Create a user."""
+        self.require_permission(identity, "create")
+        # validate data
+        data, errors = self.schema.load(
+            data,
+            context={"identity": identity},
+        )
+        # create the role with the specified data
+        group = self.record_cls.create(data)
+
+        # run components
+        self.run_components(
+            "create",
+            identity,
+            data=data,
+            group=group,
+            errors=errors,
+            uow=uow,
+        )
+
+        uow.register(RecordCommitOp(group, indexer=self.indexer, index_refresh=True))
+
+        return self.result_item(
+            self, identity, group, links_tpl=self.links_item_tpl, errors=errors
+        )
+
+    @unit_of_work()
+    def update(self, identity, id_, data, revision_id=None, uow=None, expand=False):
```

Comment:
Since we are doing schema validation here, I would add `raise_errors=True,`. Same as you did for `create`.